linux/LinuxProcess.c

```diff
@@ -362,6 +365,23 @@ static void LinuxProcess_rowWriteField(const Row* super, RichString* str, Proces
          xSnprintf(buffer, n, "N/A  ");
       }
       break;
+   #ifdef HAVE_SYSCALL
+   case SYSCALL: {
+      switch (lp->syscall_state) {
+      case SYSCALL_STATE_NORMAL:
+         xSnprintf(buffer, n, "%-8d", lp->syscall_num);
```

Comment:
Using the number here may not be very useful nor intuitive for the user. Also 8 characters is kinda short to print a readable name. Maybe use a dynamically resizeable column for display?